ui/perfherder/graphs/LegendCard.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Badge, Button, FormGroup, Input } from 'reactstrap';
+import { Badge, Button, Form } from 'react-bootstrap';
```

Comment:
<img width="326" height="373" alt="Screenshot 2025-10-07 at 16 11 38" src="https://github.com/user-attachments/assets/137d0c0f-c217-4aae-b869-09cf34c5b86b" />
Locally I see some issues with this component.